meta-oe/recipes-devtools/exprtk/exprtk_git.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "Expression parser"
+HOMEPAGE = "https://github.com/ArashPartow/exprtk"
+SECTION = "libs"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "git://github.com/ArashPartow/exprtk.git;nobranch=1"
+SRCREV = "281c2ccc65b8f91c012ea3725ebcef406378a225"
+S = "${WORKDIR}/git"
+
+# exprtk is a header only C++ library, so the main package will be empty.
+
+RDEPENDS_${PN}-dev = ""
```

Comment:
run it through meta-openembedded/contrib/oe-stylize.py script so sections are arranged correctly.

Reply:
Ack